.eslintrc.js

```diff
@@ -58,6 +58,7 @@ module.exports = {
       'error',
       { fixMixedExportsWithInlineTypeSpecifier: false },
     ],
+    'valid-jsdoc': 'off',
```

Comment:
What was the reason to disable this rule?

Reply:
`valid-jsdoc` rule shows error when `@return {type} ...` or `@param {type} ...` isn't added. However when you include it `tsdoc/syntax` rule shows error because you included the type. These rules are excluding each other